nl_server/lib/nl_embeddings.py

```diff
@@ -63,12 +64,9 @@ def __init__(self) -> None:
   def _download_embeddings(self):
     storage_client = storage.Client()
     bucket = storage_client.bucket(bucket_name=GCS_BUCKET)
-    try:
-      blob = bucket.get_blob(FOLDER + EMBEDDINGS_FILENAME)
-      # Download
-      blob.download_to_filename(os.path.join(TEMP_DIR, EMBEDDINGS_FILENAME))
-    except Exception as e:
-      logging.info(e)
+    blob = bucket.get_blob(self.gcs_folder + self.embeddings_file)
```

Comment:
Are we sure that this will never ever fail? For example, what if the file on the server is somehow deleted? Won't it make sense to keep this under the try/catch?

Reply:
Previously, it logs the failure but the program would continue. Since this is used when server starts, it is better to let it throw instead of silently proceed? Since if no model is loaded, the server should not start.

Reply:
i see, ok sounds good.